draft-jasdips-regext-rdap-rpki.md

```diff
@@ -686,6 +696,230 @@ Here is an elided example for an autonomous system number object with ASPAs:
+    }
+  ]
+}
+```
+
+## Lookup
+
+The resource type path segment for exact match lookup of a BGPSec Router Certificate object is "rpkiBgpsecRouterCert".
+
+The following lookup path segment is defined for a BGPSec Router Certificate object:
+
+Syntax: rpkiBgpsecRouterCert/<autonomous system number>
+
+For example:
+
+```
+https://example.net/rdap/rpkiBgpsecRouterCert/65536
+```
+
+## Search
+
+The resource type path segment for searching BGPSec Router Certificate objects is "rpkiBgpsecRouterCerts".
+
+The following search path segments are defined for BGPSec Router Certificate objects:
+
+Syntax: rpkiBgpsecRouterCerts?handle=<handle search pattern>
+
+Syntax: rpkiBgpsecRouterCerts?issuer=<certificate issuer search pattern>
+
+Syntax: rpkiBgpsecRouterCerts?subject=<certificate subject search pattern>
+
+Searches for BGPSec router certificate information by handle are specified using this form:
+
+rpkiBgpsecRouterCerts?handle=XXXX
+
+XXXX is a search pattern representing the "handle" property of a BGPSec Router Certificate object, as described in
+(#bgpsec_router_cert_object_class). The following URL would be used to find information for BGPSec Router Certificate
+objects with handle matching the "ABC*" pattern:
+
+```
+https://example.net/rdap/rpkiBgpsecRouterCerts?handle=ABC*
+```
+
+Searches for BGPSec router certificate information by certificate issuer are specified using this form:
+
+rpkiBgpsecRouterCerts?issuer=YYYY
+
+YYYY is a search pattern representing the "issuer" property of a BGPSec Router Certificate object, as described in
+(#bgpsec_router_cert_object_class). The following URL would be used to find information for BGPSec Router Certificate
+objects with issuer matching the "CN=ISP-C*" pattern:
+
+```
+https://example.net/rdap/rpkiBgpsecRouterCerts?issuer=CN%3DISP-C*
+```
+
+Searches for BGPSec router certificate information by certificate subject are specified using this form:
+
+rpkiBgpsecRouterCerts?subject=ZZZZ
+
+ZZZZ is a search pattern representing the "subject" property of a BGPSec Router Certificate object, as described in
+(#bgpsec_router_cert_object_class). The following URL would be used to find information for BGPSec Router Certificate
+objects with subject matching the "CN=ROUTER-ASN-655*" pattern:
+
+```
+https://example.net/rdap/rpkiBgpsecRouterCerts?subject=CN%3DROUTER-ASN-6553*
+```
+
+### Search Results
+
+The BGPSec Router Certificate search results are returned in the "rpkiBgpsecRouterCertSearchResults" member, which is an
+array of BGPSec Router Certificate objects ((#bgpsec_router_cert_object_class)).
+
+Here is an elided example of the search results when finding information for BGPSec Router Certificate objects with
+subject matching the "CN=ROUTER-ASN-655*" pattern:
+
+```
+{
+  "rdapConformance":
+  [
+    "rdap_level_0",
+    "rpki1",
+    "rpkiBgpsecRouterCert",
+    "rpkiBgpsecRouterCerts",
+    "rpkiBgpsecRouterCertSearchResults",
+    ...
+  ],
+  ...
+  "rpkiBgpsecRouterCertSearchResults":
```

Comment:
see the latest in the extensions draft on search results and extensions.

Reply:
Will change.